test/Linter/Yaml/YamlLinterTest.php

```diff
@@ -78,7 +83,12 @@ function it_should_handle_exceptions_on_invalid_type()
     {
         $this->linter->setObjectSupport(false);
         $this->linter->setExceptionOnInvalidType(true);
-        $fixture = YamlLinter::supportsFlags() ? 'object-support.yml' : 'object-support-old.yml';
+
+        $fixture = 'object-support-old.yml';
+        if (YamlLinter::supportsFlags()) {
```

Comment:
It's better to move these 3 different test scenario's to 3 different tests and skip the once that do not apply instead.